pkg/render/dex_utils.go

```diff
@@ -0,0 +1,288 @@
+// Copyright (c) 2019 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+package render
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+
+	oprv1 "github.com/tigera/operator/api/v1"
+)
+
+type ConnectorType string
+
+const (
+	ConnectorTypeOIDC      = "oidc"
```

Comment:
Are these variables specific to Dex? If yes, could you please prefix them with Dex or something like that.

Also, if these aren't used outside the package please leave them unexported.